qrenderdoc/CMakeLists.txt

```diff
@@ -309,11 +318,20 @@ if(ENABLE_QRENDERDOC)
 # which causes CMake to rerun this target every time so that Qt
 # can do dependency checking and rebuild anything necessary.
 add_custom_command(OUTPUT QRenderDoc
-    COMMAND ${QMAKE_QT5_COMMAND} "CMAKE_DIR=${CMAKE_BINARY_DIR}" ${CMAKE_CURRENT_SOURCE_DIR}
+    COMMAND ${SET_SYSTEM_PATH_COMMAND} ${QMAKE_QT5_COMMAND} "CMAKE_DIR=${CMAKE_BINARY_DIR}" ${CMAKE_CURRENT_SOURCE_DIR}
     COMMAND ${GENERATOR_MAKE} ${GENERATOR_MAKE_PARAMS}
     DEPENDS RenderDoc.icns)
 add_custom_target(build-qrenderdoc ALL DEPENDS QRenderDoc DEPENDS renderdoc DEPENDS swig-bindings)
 
+if(APPLE)
+    set_target_properties(build-qrenderdoc PROPERTIES XCODE_GENERATE_SCHEME ON)
```

Comment:
Is this line necessary? The documentation for `CMAKE_XCODE_GENERATE_SCHEME` suggests that it propagates to `XCODE_GENERATE_SCHEME` on all targets.

Reply:
I do not recall why this was here - perhaps this was done first.
I have removed it locally and the Xcode project seems to be behaving the same.

Reply:
Happy to remove this line

Reply:
Reverted line and squashed into

https://github.com/baldurk/renderdoc/pull/2287/commits/92745c6436606e0df109fcb221ab58a8eedacf7d